doc/source/analyzing/objects.rst

```diff
@@ -200,6 +200,26 @@ could supply:
 This can select both particles and mesh fields.  Mesh fields will be 3D arrays,
 and generated through volume-weighted overlap calculations.
 
+.. note::
+
+   You can now select easily specify that the resolution should be a multiple
+   of the domain resolution!
+
+By using negative complex integers you can specify that the size be a multiple
+of the underlying domain dimensions.  For instance, if you don't know the size
+of the dataset's base dimensions, you can specify ``-1j`` as the step in a slice
+and get back the native coordinates.
```

Comment:
This is not clear to me. What if my grid is AMR in that direction ? Do I get to keep the native (varying) resolution, or does everything get interpolated to the lowest (base) resolution ?